src/xcode/ENA/ENA/Source/Scenes/Home/HomeInteractor.swift

```diff
@@ -218,14 +218,23 @@ extension HomeInteractor {
 			)
 			inactiveConfigurator?.activeAction = inActiveCellActionHandler
 		case .unknownOutdated:
-			riskLevelConfigurator = HomeUnknown48hRiskCellConfigurator(
-				isLoading: isRequestRiskRunning,
-				lastUpdateDate: dateLastExposureDetection,
-				detectionInterval: detectionInterval,
-				detectionMode: detectionMode,
-				manualExposureDetectionState: riskProvider.manualExposureDetectionState,
-				previousRiskLevel: store.previousRiskLevel)
-			inactiveConfigurator?.activeAction = inActiveCellActionHandler
+			if detectionMode == .automatic {
+				inactiveConfigurator = HomeInactiveRiskCellConfigurator(
+					inactiveType: .outdatedResults,
+					previousRiskLevel: store.previousRiskLevel,
+					lastUpdateDate: dateLastExposureDetection
+				)
+				inactiveConfigurator?.activeAction = inActiveCellActionHandler
+			} else {
+				riskLevelConfigurator = HomeUnknown48hRiskCellConfigurator(
+					isLoading: isRequestRiskRunning,
+					lastUpdateDate: dateLastExposureDetection,
+					detectionInterval: detectionInterval,
+					detectionMode: detectionMode,
+					manualExposureDetectionState: riskProvider.manualExposureDetectionState,
+					previousRiskLevel: store.previousRiskLevel)
+				inactiveConfigurator?.activeAction = inActiveCellActionHandler
```

Comment:
Is this line needed? It looks like, in this case riskLevelConfigurator is returned and changing inactiveConfigurator has no effect.

Reply:
You're right, I remove this line of code